src/cascadia/TerminalSettingsModel/TerminalSettings.idl

```diff
@@ -28,12 +28,15 @@ namespace Microsoft.Terminal.Settings.Model
 
         static TerminalSettingsCreateResult CreateWithProfile(CascadiaSettings appSettings, Profile profile, Microsoft.Terminal.Control.IKeyBindings keybindings);
         static TerminalSettingsCreateResult CreateWithNewTerminalArgs(CascadiaSettings appSettings, NewTerminalArgs newTerminalArgs, Microsoft.Terminal.Control.IKeyBindings keybindings);
-        static TerminalSettingsCreateResult CreateWithParent(TerminalSettingsCreateResult parent);
 
         void SetParent(TerminalSettings parent);

```

Comment:
So, we've removed `CreateWithParent`, but `TerminalSettings` can still get/set a parent. Is that going to be a problem? Like, could we theoretically rely on that parent/inheritance and the cross-model stuff will break? Or is that not a problem because you just have to make sure the inheritance stuff stays out of the `TermControl` and `TermApp` layers?

Reply:
oh ho ho, no one's calling that anymore. Gonna delete that too. Knew I forgot something in here!